README.md

```diff
@@ -1,2 +1,183 @@
+
+Also, [create a new database](docs/images/Step1-CreateANewDatabase.png) if you don't have one.
+
+### Step 2: Create Cloudant Package Binding - TrackPRsInCloudant
+
+Get the Cloudant [Service Credentials](https://github.com/IBM-Bluemix/docs/blob/master/services/Cloudant/tutorials/images/img0009.png) from Step 1.
+
+
+```bash
+cat cloudant.json
+{
+  "username": "<username>",
+  "password": "<password>",
+  "host": "<host>",
+  "port": <port>,
+  "url": "<url>",
+  "dbname": "<dbname>"
+}
+wsk package bind /whisk.system/cloudant TrackPRsInCloudant --param-file cloudant.json
+```
+
+This will result in a cloudant package and you can verify by running a sample action:
+
+```bash
+wsk action invoke TrackPRsInCloudant/list-documents --blocking
+```
+
+### Step 3: Create an Action - track-pull-requests
+
+```bash
+git clone git@github.ibm.com:pdesai/GitHubSlackBot.git
+cd GitHubSlackBot
+wsk action create track-pull-requests openwhisk/actions/js/track-pull-requests.js --param cloudant_package TrackPRsInCloudant
+```
+
+### Step 4: Create GitHub WebHook trigger - GitHubWebHookTrigger
+
+ 1. Generate GitHub [Personal Access Token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).
+ 2. Create a GitHub Package Binding:
+
+ ```bash
+ cat github.json
+ {
+  "username": "<username>",
+  "repository": "<repository>",
+  "accessToken": "<accessToken>"
+}
+ wsk package bind /whisk.system/github GitHubWebHook --param-file github.json
+ ```
+ 
+ 3. Create a WebHook trigger for the GitHub **pull-requests** by using <package_name>/webhook feed created in Step 2:
+
+ ```bash
+ wsk trigger create GitHubWebHookTrigger --feed GitHubWebHook/webhook --param events pull-requests
+ ```
+
+ 4. This should create a webhook under GitHub repository, for example:
+  
+ ![WebHook Settings](docs/images/Step4-WebHooksSettings.png "WebHook Settings")
+
+### Step 5: Create a Rule - RuleToTrackPullRequests
+
+```bash
+wsk rule create RuleToTrackPullRequests /<namespace>/GitHubWebHookTrigger /<namespace>/track-pull-requests
+```
+
+At this point, any new pull request update is being recorded in Cloudant database, for example:
+
+![Cloudant Database](docs/images/Step5-PullRequestsInDatabase.png)
+
+### Step 6: Create an Action - find-delayed-pull-requests
+
+```bash
+cat find-delayed-pull-requests.json
+{
+  "cloudant_package": "TrackPRsInCloudant",
+  "github_username": "<github_username>",
+  "github_access_token": "<github_access_token>"
+}
+wsk action create find-delayed-pull-requests openwhisk/actions/js/find-delayed-pull-requests.js --param-file find-delayed-pull-requests.json
```

Comment:
Add a step to invoke this action manually. Response should look something like:

```json
"response": {
        "result": {
            "prs": []
        },
        "status": "success",
        "success": true
    }
```

This will give the user some indication that they completed this step successfully.